### PR TITLE
Matmul sweep

### PR DESCRIPTION
**Pull request by andrej**
So far for the whole array reference design, should be easy to port similarly to the other designs

### CODE REVIEW

**Review by jgmelber — APPROVED** [review]

File: reference_designs/ipu-xrt/matrix_multiplication.h [hunk 1]
```diff
@@ -47,7 +47,14 @@ void add_default_options(po::options_description &desc) {
       "verbosity,v", po::value<int>()->default_value(0),
       "the verbosity of the output")(
       "instr,i", po::value<std::string>()->required(),
-      "path of file containing userspace instructions to be sent to the LX6");
+      "path of file containing userspace instructions to be sent to the LX6")(
```
Comment (jgmelber) [inline reference_designs/ipu-xrt/matrix_multiplication.h:50]:
```suggestion
      "path of file containing userspace instructions sent to the NPU")(
```

**Comment by jgmelber** [comment]
https://github.com/psf/black?tab=readme-ov-file#usage

**Comment by jgmelber** [comment]
> What command should I use to reformat the Python code?

https://github.com/psf/black?tab=readme-ov-file#usage

**Comment by andrej** [comment]
Sorry Joe, wanted to squeeze in these last minute changes to the plotter. Feel free to merge now once these checks pass.

**Pull request merged**